packages/manager/modules/dbaas-logs/src/logs/detail/detail.constants.js

```diff
@@ -21,16 +21,22 @@ export const GUIDES = {
   SG: 'https://docs.ovh.com/sg/en/',
   WE: 'https://docs.ovh.com/us/en/',
   WS: 'https://docs.ovh.com/us/es/',
-  US: 'https://docs.us.ovhcloud.com',
+  US: 'https://support.us.ovhcloud.com/hc/en-us',
   IN: 'https://docs.ovh.com/asia/en/',
 };
 
+export const LOG_DATA_PLATFORM_GUIDES = {
+  US:
+    'https://support.us.ovhcloud.com/hc/en-us/sections/27230973429523-Logs-Data-Platform',
+};
+
```

Comment:
Links for other subs missing.

Reply:
for other subs, it's managed dynamically by the doc-resolver 2API